andela/noldor-ah-backend
# #161872773 Descriptive Response When Searching

#### What does this PR do?
- Fixes the lack of a success status on successful search queries
- Fixes the lack of a descriptive message on successful search queries

#### Description of Task to be completed?
- Add `success: true` to a successful search query
- Add `message: articles matching that search term found` to a successful search query

#### How should this be manually tested?
- Make a search query #13
- Notice that the response has been changed from an array to an object
- Notice the addition of two new properties to the search response

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#161872773

#### Screenshots (if appropriate)
<img width="1128" alt="screen shot 2018-11-12 at 9 09 46 am" src="https://user-images.githubusercontent.com/36507972/48334768-0c260c80-e65c-11e8-99d4-3cdaa38c40c6.png">

#### Questions:
N/A
